rxredux/build.gradle.kts

```diff
@@ -66,7 +66,7 @@ val sourceSets = project.the<SourceSetContainer>()
 
 val sourcesJar by tasks.creating(Jar::class) {
     from(sourceSets["main"].allSource)
-    classifier = "sources"
```

Comment:
`classifier` is deprecated and recommended to use `archiveClassifier.set` instead.